.evg.yml

```diff
@@ -238,26 +246,28 @@ functions:
     - command: shell.exec
       type: test
       params:
+        silent: true
+        env:
+          PLAIN_USER: "${plain_auth_username}"
+          PLAIN_PASSWD: "${plain_auth_password}"
         script: |
           ${PREPARE_SHELL}
-          export PLAIN_USER=${plain_auth_username}
-          export PLAIN_PASSWD='${plain_auth_password}'
           ${SCRIPT_DIR}/run-integration-tests-plain.sh
 
   "run scram integration tests":
     - command: shell.exec
       type: test
       params:
+        silent: true
+        env:
+          USERNAME: "${var_username}"
+          PASSWORD: "${var_password}"
+          MECHANISM: "${var_auth_mechanism}"
+          MONGO_CLIENT_ARGS: "-u bob -p pwd123 --authenticationDatabase admin"
+          MONGO_OTHER_USER_ROLES: "[{role: 'root', db: 'admin'}]"
         script: |
           ${PREPARE_SHELL}
-          export USERNAME=bob
```

Comment:
While you've removed these from the logs, they are now in the evergreen file which is public ;)

That said, I'm not worried about these local passwords and user names.

Reply:
yes, that's correct. They were already in that file however. I am not worried about it either. I didn't create an evg variable because we disabled the project and these are "bogus" username and password for a BIC which doesn't exist or is recreated each time we run tests anyway. But I can add them if you want.